tests/hf/test_convert.py

```diff
@@ -0,0 +1,540 @@
+                ((0, "Y"), (1, "X"), (3, "Y"), (4, "Y"), (5, "Y")): (0.011149373109704066 + 0j),
+                ((0, "Z"), (1, "Z"), (2, "Z")): (0.18221934159881317 + 0j),
+                ((0, "Z"), (1, "X"), (2, "Z"), (3, "Y"), (5, "Y")): (0.029077593893863385 + 0j),
+                ((1, "Y"), (2, "Z"), (3, "Y"), (4, "Z"), (5, "X")): (-0.029077593893863385 + 0j),
+                ((0, "Z"), (1, "Y"), (2, "X"), (3, "X"), (4, "Y"), (5, "X")): (
+                    0.00043511791324680484 + 0j
+                ),
+                ((0, "Z"), (1, "Y"), (2, "Y"), (3, "X"), (4, "X"), (5, "X")): (
+                    -0.00043511791324680484 + 0j
+                ),
+                ((1, "Y"), (2, "Y"), (3, "Y"), (4, "Y"), (5, "X")): (0.00043511791324680484 + 0j),
+                ((1, "Y"), (2, "X"), (3, "Y"), (4, "X"), (5, "X")): (0.00043511791324680484 + 0j),
+                ((0, "Z"), (2, "Z"), (3, "Z")): (0.16824174504299702 + 0j),
+                ((0, "Z"), (1, "Y"), (2, "Z"), (3, "X"), (5, "Y")): (0.029512711807110188 + 0j),
+                ((1, "X"), (2, "Z"), (3, "X"), (4, "Z"), (5, "X")): (0.029512711807110188 + 0j),
+                ((0, "Z"), (1, "Z"), (4, "Z")): (0.13726839006899005 + 0j),
+                ((0, "Z"), (1, "X"), (3, "Y"), (4, "Z"), (5, "Y")): (0.011149373109704066 + 0j),
+                ((1, "Y"), (3, "Y"), (5, "X")): (-0.011149373109704066 + 0j),
+                ((0, "Z"), (1, "Z"), (4, "Z"), (5, "Z")): (0.12008313883007578 + 0j),
+                ((1, "Z"), (3, "Z")): (0.22003977334376118 + 0j),
+                ((2, "Y"), (4, "Y"), (5, "Z")): (0.009647475282106617 + 0j),
+                ((1, "Z"), (2, "X"), (3, "Z"), (4, "X")): (0.009647475282106617 + 0j),
+                ((2, "X"), (4, "X"), (5, "Z")): (0.009647475282106617 + 0j),
+                ((1, "Z"), (2, "Y"), (3, "Z"), (4, "Y")): (0.009647475282106617 + 0j),
+                ((2, "Z"), (4, "Z")): (0.13758959215600186 + 0j),
+                ((2, "Z"), (4, "Z"), (5, "Z")): (0.1472370674381085 + 0j),
+                ((1, "Z"), (2, "Z"), (3, "Z"), (4, "Z")): (0.1472370674381085 + 0j),
+                ((1, "Z"), (2, "Z"), (3, "Z"), (4, "Z"), (5, "Z")): (0.13758959215600186 + 0j),
+                ((5, "Z"),): (0.1492827559305538 + 0j),
+            },
+        ),
+    ],
+)
+def test_observable_conversion(mol_name, terms_ref, custom_wires, monkeypatch):
+    r"""Test the correctness of the QubitOperator observable conversion from
+    OpenFermion to Pennylane.
+
+    The parametrized inputs are `.terms` attribute of the output `QubitOperator`s based on
+    the same set of test molecules as `test_gen_hamiltonian_pauli_basis`.
+
+    The equality checking is implemented in the `convert` module itself as it could be
+    something useful to the users as well.
+    """
+    qOp = QubitOperator()
+    if terms_ref is not None:
+        monkeypatch.setattr(qOp, "terms", terms_ref)
```

Comment:
I am not sure why monkeypatch is used. Why not building the observable with openfermion directly.

Reply:
maybe @agran2018 knows?

Reply:
Can't remember if this is from me or Alain, but the reason is likely wanting to unit test only the function rather than testing also the construction of openfermion observable.